packages/shared/forks/ReactFeatureFlags.test-renderer.js

```diff
@@ -36,7 +36,7 @@ export const disableTextareaChildren = false;
 export const enableSuspenseAvoidThisFallback = false;
 export const enableSuspenseAvoidThisFallbackFizz = false;
 export const enableCPUSuspense = false;
-export const enableUseMemoCacheHook = true;
+export const enableUseMemoCacheHook = __EXPERIMENTAL__;
```

Comment:
This needs to match as a bundle test is using the test renderer and asserting this isn't available in non-experimental.